### PR TITLE
Updates and fixes to cdn-improve-performance

### PR DESCRIPTION
**Pull request by Lichard**
Change 128 bytes to 1 byte.
Updated intro paragraph
Updated table wording

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Lichard__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: articles/cdn/cdn-improve-performance.md [hunk 1]
```diff
@@ -89,35 +89,35 @@ There are two ways CDN can support compression:
 		
 > [AZURE.IMPORTANT] To be eligible for compression, a file must:
 >
-> - Be larger than 128 bytes.
+> - Be larger than 1 byte.
```
Comment (CamSoper) [inline articles/cdn/cdn-improve-performance.md]: Are you *sure* about this difference?  One of the docs you sent me said 1 byte, the other said 128 bytes.  It didn't make sense to me that we'd compress a 1 byte file, so I did a test with a 19 B file and a 2.2 KB file on the same endpoint.  The 19 B file came back uncompressed, while the 2.2 KB file *was* compressed.  I'd give you the URLs except that endpoint is in a broken state right now and I'm waiting for some settings to propagate.

**Comment by CamSoper** [comment]
Okay, I just checked the Edgecast doc.  It says 1 byte, but that seems odd.  Why even specify 1 byte?  By definition, a file can't be any smaller.

**Comment by CamSoper** [comment]
Okay, I was able to run my test again.  Everything is going against my endpoint named `cdndocdemo`.

| Request | Original size | Compressed |
|-----------|-----------------|-----------------|
| /publicblob/test.txt | 19 B | No |
| /publicblob/lorem-64b.txt | 82 B | No |
| /publicblob/lorem-128b.txt | 142 B | No |
| /publicblob/lorem-256b.txt | 268 B | No |
| /publicblob/lorem-288b.txt | 290 B | No |
| /publicblob/lorem-300b.txt | 300 B | No |
| /publicblob/lorem-301b.txt | 301 B | **Yes** |
| /publicblob/lorem-320b.txt | 327 B | **Yes** |
| /publicblob/lorem-384b.txt | 398 B | **Yes** |
| /publicblob/lorem-512b.txt | 515 B | **Yes** |
| /publicblob/lorem-1kb.txt | 1.1 KB | **Yes** |

So it would appear the minimum size is 301 B.  Seems arbitrary, but there it is.  Feel free to replicate my test.  :)

**Comment by CamSoper** [comment]
Adding @zhangmanling to this conversation so we don't have to keep switching between email and GitHub. 

I was wrong.  Looks like it **is** in the 128 B range.

**Comment by Lichard** [comment]
Seems like the EC docs are wrong then. I'll modify the commit back to 128B for now, and will bring it up with them.

**Comment by CamSoper** [comment]
@ggailey777 - Please go ahead and close this PR.  @Lichard is a program manager in my product area, so we should get him on the private repo.

**Comment by ggailey777** [comment]
Closed per request.

**Pull request closed without merging**